systemd-units/gpg-agent@.service

```diff
@@ -1,11 +1,12 @@
 [Unit]
-Description=gpg4win to WSL connector for GPG
+Description=gpg4win to WSL connector for GPG (S.gpg-agent)
 Requires=gpg-agent.socket
 
 [Service]
 Type=simple
-ExecStart=%h/.local/bin/npiperelay.exe -ep -ei -s -a 'C:/Users/alexm/AppData/Local/gnupg/S.gpg-agent.extra'
+ExecStart=%h/.local/bin/npiperelay.exe -v -p -ep -ei -s -a 'C:/Users/alexm/AppData/Local/gnupg/S.gpg-agent'
```

Comment:
Is there a way to make the `C:/Users/alexm/` (`~`) dynamic?

Reply:
While it **might** be possible to use `%USERPROFILE%` it would also have no purpose since WSL installations are always specific to the Windows user. If you log in as a different Windows user you'll have a different WSL install, so having a hardcoded path to your user account isn't a problem.

Also, when you run the installer (or follow the manual instructions in the readme), part of the process involves updating the path there, so it's dynamically updated during the install process.